Sprint_Tests/Yehor_Fedosieiev.md

```diff
@@ -0,0 +1,25 @@
+#Test №1
+
+Раньше он хорошо разбирался в верстке и работе на передовых технологиях на фронте. После небольшого перерыва в несколько лет и глубокой депрессии плюс прослушивания тяжелой музыки, он слабо разбирается в современных тенденциях и много пьет. Если бы он справился с депрессией после смерти жены и ребенка, у него бы сейчас не стоял вопрос поиска работы с релевантной зарплатой.
+
+>He used to be good at Front-end and work via top-notch front-end technologies. After a short break for several years and deep depression as well as listening to hard-music, he has a low level of modern trends and drinks a lot. If he had got through the depression caused by his wife’s and kid’s death, he wouldn’t be looking for a job with a relevant salary.
+
+Будь у него цель и адекватные зарплатные ожидания он бы уже работал на удаленке где нибудь в штатах или на Бали с перспективой перебраться на Канарские острова и оформить испанское гражданство. Он с детства мечтал жениться на испанке и рано выйти на пенсию и разводить виноград на побережье Балеарского моря.
+
+>If he had a goal and an adequate salary expectation, he would have been working remotely somewhere in the USA or in Bali with a possibility to relocate to the Canary Islands and apply for Spanish nationality. He has dreamt of getting married on Spanish girl, retiring early and planting/growing grapes on the coast of the Balearic sea since his childhood.
+
+Если не знаешь как работает кусок кода, написанный коллегой тебе придется покрыть его консоль логами и проследить всю логическую цепочку. Это трудоемкое занятие. Если бы у нас была документация к этой фиче или хотя бы тикет на реализацию фичи с описанием бизнес логики, мы бы вылили фикс в течении пары часов.
+
+>If you don’t know how a piece of code written by your peer works, you will have to cover his console with logs and watch the whole logic. It is a labour-intensive/time-consuming task. If we had a documentation for the feature or at least a ticket to implement the feature including a business-logic description, we would implement a fix within a few hours.
+
+Я хочу чтобы вы не стеснялись выражать свои мысли на иностранном языке не взирая на ошибки, которые могут появиться ввиду отсутствия практики и нерегулярности занятий языком. Я хочу чтобы каждый из вас до конца года прочел по книге в оригинале, язык оригинала естественно английский, и получил удовольствие от преодоления языкового барьера в общении.
+
+>I want you not to hesitate to express your own ideas using a foreign language despite mistakes which might appear because of a lack of practise and studying the language on and off / infrequency of studying. I want each of you to read a book in the original language, obviously it is English, and to enjoy transcending a language barrier.
+
```

Comment:
in a foreign language
despite the mistakes (you have description of what kind of mistakes may arise)
lack of practice (noun)
lack of regularity in studying the language
obviously in English
overcoming a language barrier
You have missed an interesting point inthis sentence, it says by the end of the year thus a more complex structure can be used

Reply:
I want each of you to have read a book by the end of the year ...